src/main/java/net/dries007/tfc/client/model/MoldsModelLoader.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed under the EUPL, Version 1.2.
+ * You may obtain a copy of the Licence at:
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ */
+
+package net.dries007.tfc.client.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.joml.Vector3f;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonSyntaxException;
+
+import net.dries007.tfc.util.Helpers;
+import net.minecraft.client.renderer.block.model.BlockElement;
+import net.minecraft.client.renderer.block.model.BlockElementFace;
+import net.minecraft.client.renderer.block.model.BlockFaceUV;
+import net.minecraft.core.Direction;
+import net.minecraft.util.GsonHelper;
+import net.neoforged.neoforge.client.model.ElementsModel;
+import net.neoforged.neoforge.client.model.geometry.IGeometryLoader;
+
+public class MoldsModelLoader implements IGeometryLoader<ElementsModel> {
```

Comment:
Braces on newlines, please apply the TFC code style xml. Applies to all java files.